### PR TITLE
Update Component section of OpenAPI documentation

### PR DESCRIPTION
**Pull request by hkhattabii**
**Is your feature request related to a problem? Please describe.**
Update Component section of OpenAPI documentation

**ID**

### CODE REVIEW

File: src/paths/components/create.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+summary: Post a new component
```
Comment (BerrevoetsRobbe) [inline src/paths/components/create.yaml:1]: rename to 'Create a new component'

File: src/paths/components/create.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    content:
+      application/json:
+        schema:
+          $ref: "../../schemas/response/ErrorResponse.yaml"
+        example:
+          timestamp: "2021-02-09T07:57:01.062+0000"
+          status: 403
+          error: "Forbidden"
+          message: "Access denied"
+          path: "/api/components/"
```
Comment (BerrevoetsRobbe) [inline src/paths/components/create.yaml:25]: This response is often returned, can we extract this to a default response?

File: src/paths/components/create.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  '404':
+    description: Expected response to an existing component
```
Comment (BerrevoetsRobbe) [inline src/paths/components/create.yaml:27]: existing -> non-existing

File: src/schemas/scriptExecutions/ActionExecutionDto.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    type: boolean
+  errorExpected:
+    type: boolean
+  status:
+    type: string
+    enum: ["RUNNING", "SUCCESS", "WARNING", "ERROR", "STOPPED", "SKIPPED"]
+  startTimestamp:
+    type: string
+  endTimestamp:
+    type: string
```
Comment (BerrevoetsRobbe) [inline src/schemas/scriptExecutions/ActionExecutionDto.yaml:26]: Can you mark these as 'timestamps'?

File: src/schemas/scriptExecutions/ScriptExecutionDto.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    type: string
+  environment:
+    type: string
+  status:
+    type: string
+    enum: ["RUNNING", "SUCCESS", "WARNING", "ERROR", "STOPPED", "SKIPPED"]
+  startTimestamp:
+    type: string
+  endTimestamp:
+    type: string
```
Comment (BerrevoetsRobbe) [inline src/schemas/scriptExecutions/ScriptExecutionDto.yaml:28]: Can you mark these as 'timestamps'?

File: src/schemas/executionRequests/ExecutionRequestDto.yaml [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+    type: [NEW, SUBMITTED, ACCEPTED, DECLINED, STOPPED, COMPLETED, KILLED, UNKNOWN]
+  scriptExecutionRequests:
+    type: array
+    items:
+      $ref: "./ScriptExecutionRequestDto.yaml"
+  executionRequestLabels:
+    type: array
+    items:
+      $ref: "./ExecutionRequestLabelDto.yaml"
+
```
Comment (BerrevoetsRobbe) [inline src/schemas/executionRequests/ExecutionRequestDto.yaml:25]: missing request timestamp

File: src/schemas/scriptExecutions/ActionExecutionDto.yaml (hunk 4 above)
Comment (hkhattabii) [reply src/schemas/scriptExecutions/ActionExecutionDto.yaml:26]: Do you mean I have to change the format into "date-time" ? The variable names are the same as the class defined in the rest-server

Comment (BerrevoetsRobbe) [reply src/schemas/scriptExecutions/ActionExecutionDto.yaml:26]: Indeed, you just mark the the format as 'date-time' (https://swagger.io/docs/specification/data-models/data-types/#string)

Comment (hkhattabii) [reply src/schemas/scriptExecutions/ActionExecutionDto.yaml:26]: Got it

File: src/schemas/scriptExecutions/ScriptExecutionDto.yaml (hunk 5 above)
Comment (BerrevoetsRobbe) [reply src/schemas/scriptExecutions/ScriptExecutionDto.yaml:28]: forgot this one ;)

Comment (hkhattabii) [reply src/schemas/scriptExecutions/ScriptExecutionDto.yaml]: Voila :)

**Review by BerrevoetsRobbe — COMMENTED** [review]
approved

**Pull request merged**